.utils/addon_specific_templates/examples/simple/variables.tf.tpl

```diff
@@ -0,0 +1,36 @@
+# TEMPLATE: This file was automatically generated with `generate_addon_structure.sh`
+# TEMPLATE: and should be modified as necessary
+# TEMPLATE:
+# TEMPLATE: All variables must have a description and should declare their type.
+# TEMPLATE: Set defaults whenever possible but do not set defaults for required properties.
+# TEMPLATE: Declare all variables in this file, sprawling declarations are difficult to identify.
+# TEMPLATE:
+# TEMPLATE: https://www.terraform.io/docs/language/values/variables.html
+# TEMPLATE: https://www.terraform.io/docs/language/expressions/types.html
+#
+variable "auth_token" {
```

Comment:
What about not using the TF_VAR in front of it, so folks can just export $(metal env) ?

Reply:
`eval $(metal env)` already exists.

Reply:
https://github.com/equinix-labs/terraform-equinix-kubernetes-addons/blob/8e7ce00af6d2b0379a4ed859c192b82fcc73a4e2/.utils/addon_specific_templates/examples/simple/variables.tf.tpl#L11-L20

I've replaced them with "metal_auth_token" and "metal_project_id" for me it makes sense, and in fact for metal_auth_token it will also be possible to define METAL_AUTH_TOKEN instead of TF_VAR_metal_auth_token 

https://registry.terraform.io/providers/equinix/equinix/latest/docs#:~:text=This%20argument%20can%20also%20be%20specified%20with%20the%20EQUINIX_API_TOKEN%20shell%20environment%20variable.

Reply:
I think `TF_VAR_metal_auth_token` is the var name

Reply:
Oh, I see I put it wrong in the description. I'll update it